src/components/Icon/Icon.tsx

```diff
@@ -4,38 +4,155 @@ import React from "react";
 
 import type { ClassName, PropsWithSpread, ValueOf } from "types";
 
-export const ICONS = {
+export const STANDART_ICONS = {
   anchor: "anchor",
-  chevronDown: "chevron-down",
-  chevronUp: "chevron-up",
+  "chevron-down": "chevron-down",
```

Comment:
Any reason for changing the values to use dashes?

As far as I remember we mapped those to camelCase for a more natural usage in React (not to be restricted by CSS class naming).

Reply:
I agree. Here the choice is between the same as vanillaframework or a more familiar format for React. I'll change it.